ocs_ci/ocs/must_gather/const_must_gather.py

```diff
@@ -694,7 +694,6 @@
     "noobaa-db-pg-0.yaml",
     "db-noobaa-db-pg-0.yaml",
     "noobaa-db-pg-0.log",
-    "noobaa-ceph-objectstore-user.yaml",
```

Comment:
"noobaa-ceph-objectstore-user.yaml" will only appear on on-prem..
We dont test must-gather according to platform